src/theme/basic/index.scss

```diff
@@ -138,6 +155,73 @@ $group-summary: #007fa3;
         overflow: visible;
     }
 
+    .q-grid-icon-button[disabled][disabled] {
+        color: $icon-button-disabled;
+    }
+
+    .q-grid-pager {
+        font-family: Roboto, "Helvetica Neue", sans-serif;
+        padding: 11px 0;
+        height: 24px;
+        font-size: 12px;
+        color: $pager-primary;
+
+        .pager-select {
+            height: 32px;
+            padding: 3px 0;
+
+            select {
+                font-family: Roboto, "Helvetica Neue", sans-serif;
+                font-size: 12px;
+                color: $pager-primary;
+                border: none;
+                height: 24px;
+                width: 34px;
+                margin: 0 0 0 12px;
+            }
+            select:focus {
+                outline: none;
+            }
+        }
+
+        .q-grid-page {
+            margin-left: 30px;
+            margin-right: 0;
+        }
+    }
+
+    .q-grid-pager button {
+        background: transparent;
+        border: none;
+        font-size: 20px;
+        width: 24px;
+        height: 24px;
+        line-height: 24px;
+        padding-top: 2px;
+
+        &.q-grid-prev-page {
```

Comment:
lets make q-grid-pager prefix

Reply:
The same caveat as I described in my previous comment.

Reply:
done in my last commit.